### PR TITLE
workaround Firefox CORS issue

### PR DESCRIPTION
**Pull request by troggy**
Addresses #47

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "license": "GPLv3",
   "private": true,
   "scripts": {
+    "postinstall": "rm -f node_modules/web3/index.d.ts && rm -f node_modules/web3/types.d.ts",
```
Comment (troggy) [inline package.json]: @sunify FYI this is the "fix" for web3 typedefs

**Comment by troggy** [comment]
there were conflict in typedefs. Types for web3 were defined in two places: web3 package, @types/web3 package. Even if you want to use @types/web3 only, the former are still used in imports — that was the problem. To fix I'm removing typedefs from web3 package in `postinstall` script

**Comment by johannbarbie** [comment]
don't know if it is related, for me in firefox developer edition (65.0b7 ) the `Prev` button in the explorer does nothing.

**Comment by troggy** [comment]
@johannbarbie I don't think it is related. Will check anyway

**Comment by sunify** [comment]
Nice 👌

**Pull request merged**